### ISSUE TITLE
No rule to make target `games.mak'.  Stop.

### ISSUE THREAD

**Issue by Ethorbit** [ValveSoftware/source-sdk-2013#420]
Using linux abuntu I did
"make -f games.mak"
as Valve said to do:
https://developer.valvesoftware.com/wiki/Source_SDK_2013#Linux

However there IS NO games.mak file, can someone provide me with this? Am I doing something wrong?
I know this has already been an issue, it's been closed without the fix details and the person was using mac so whatever helped him/her won't help me.
https://github.com/ValveSoftware/source-sdk-2013/issues/256

Also in that post he/she did mp/src/creategameprojects
And if I run creategameprojects in the terminal it ALWAYS says invalid command.

**Comment by kisak-valve** [comment]
Hello @Ethorbit, what is the exact error you are getting? Do you have a sane gcc build environment setup? (from https://github.com/ValveSoftware/source-sdk-2013/issues/264#issuecomment-42691380)

**Comment by Ethorbit** [comment]
This is the mod I'm trying to compile:
https://github.com/Ethorbit/HL2DM-ZombiesatPhoneburnia-SDK-Latest
And these are the tools I've installed that Valve told me to get:

GCC and G++ 4.2.x or lower
Vprojtomake 2010 or Xerces XML parser 2.8.x
libstdc++ 6
libc 2.4 or higher

And I already included my exact error in my post:

> make: games.mak: No such file or directory
make: *** No rule to make target `games.mak'.  Stop.

**Comment by psychonic** [comment]
You are running into one of multiple issues.

GCC/G++ 4.2, vprojtomake, and Xerces are all related to the Source SDK 2007 version. Your mod is based on the 2013 SDK which uses a much different toolset.

One reason that your build won't get off the ground is being vpc and other binaries in the repo lost their executable bit.

Even then, building the full games.mak would also try to compile the client, which some of your modifications break. Since it seems that you're only wanting to compile the server piece, to essentially have a modded hl2dm server, you don't need to also be trying to compile the client.

I posted in your thread here with exact steps that will work to compile your mod's server.so on Linux. https://forums.alliedmods.net/showthread.php?t=296695

**Comment by Ethorbit** [comment]
Yeah I kind of replied to this person before I went to the sourcemod section, I'm trying your steps, I believe this will work in which case thank you so much.

**Comment by Ethorbit** [comment]
`Edit/Delete Message  Reply With Quote  Multi-Quote This Message  Quick reply to this message #3
---- LINKING obj_server_hl2mp_linux32/release/server.so [release] ----
true -signvalve ./obj_server_hl2mp_linux32/release/server.so;
chmod: cannot access `../../../game/mod_hl2mp/bin/server.so': No such file or directory
---- COPYING TO ../../../game/mod_hl2mp/bin/server.so [release] ----
chmod: cannot access `../../../game/mod_hl2mp/bin/server.so': No such file or directory
chmod: cannot access `../../../game/mod_hl2mp/bin/server.so.dbg': No such file or directory
/bin/sh: 1: ../../devtools/gendbg.sh: Permission denied
make: *** [../../../game/mod_hl2mp/bin/server.so] Error 126`
